test/gst-msdk/vpp/csc.py

```diff
@@ -0,0 +1,48 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+
+spec = load_test_spec("vpp", "csc")
+
+@slash.requires(have_gst)
+@slash.requires(*have_gst_element("msdk"))
+@slash.requires(*have_gst_element("msdkvpp"))
+@slash.requires(*have_gst_element("checksumsink2"))
+@slash.requires(using_compatible_driver)
+@slash.parametrize(*gen_vpp_csc_parameters(spec))
+@platform_tags(VPP_PLATFORMS)
+def test_default(case, csc):
+  params = spec[case].copy()
+  params.update(
+    csc = csc, mcscu = mapformatu(csc),
+    mformat = mapformat(params["format"]))
+  params["ofile"] = get_media()._test_artifact(
+    "{}_{format}_csc_{csc}_{width}x{height}"
+    ".yuv".format(case, **params))
+
+  call(
+    "gst-launch-1.0 -vf filesrc location={source} num-buffers={frames}"
+    " ! rawvideoparse format={mformat} width={width} height={height}"
+    " ! msdkvpp hardware=true ! video/x-raw,format={mcscu}"
+    " ! checksumsink2 file-checksum=false frame-checksum=false"
```

Comment:
checksumsink2 only supports `format: { (string)NV12, (string)I420, (string)YV12, (string)AYUV, (string)YUY2, (string)P010_10LE }`.  Please submit a patch to https://github.com/ceyusa/gst-checksumsink to add ARGB support.

Reply:
please refer to https://github.com/ceyusa/gst-checksumsink/pull/4